docs/spelling_wordlist

```diff
@@ -789,3 +790,4 @@ xgettext
 XSS
 xxxxx
 Zope
+à
```

Comment:
Is this file automatically ordered? These changes seem out of scope otherwise.

Reply:
I automatically ordered it in my editor, but apparently that was a Bad Idea. I restored the old ordering instead so the diff is less polluted.